components/cam/bld/configure

```diff
@@ -971,6 +978,24 @@ my $clubb_do_adv = $cfg_ref->get('clubb_do_adv');
 if ($print>=2) { print "clubb_do_deep=',$clubb_do_deep,$eol"; }
 if ($print>=2) { print "clubb_do_adv=',$clubb_do_adv,$eol"; }
 
+#-----------------------------------------------------------------------------------------------
+# SHOC
+
+if (defined $opts{'shoc_sgs'}) {
+    $cfg_ref->set('shoc_sgs', $opts{'shoc_sgs'});
+}
+my $shoc_sgs = $cfg_ref->get('shoc_sgs');
+
+# consistency checks...
+
+# SHOC_SGS only works with mg microphysics
+if ($shoc_sgs and $microphys_pkg !~ m/^mg/) {
+    die <<"EOF";
+**    ERROR: microphysics package set to: $microphys_pkg
+**    SHOC only works with MG microphysics.
+EOF
+}
+
```

Comment:
I didn't see an option for P3 for the "microphys_pkg".  For now I'll go ahead and delete this check so that SHOC won't bomb out when we try to run it with P3